include/power_grid_model/component/transformer_utils.hpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+#ifndef POWER_GRID_MODEL_COMPONENT_TRANSFORMER_UTILS_HPP
+#define POWER_GRID_MODEL_COMPONENT_TRANSFORMER_UTILS_HPP
+
+inline double tap_adjust_impedance(const double& tap_pos, const double& tap_min, const double& tap_max,
```

Comment:
@nitbharambe for small data type like `double`. Just pass by value, no need by const reference